### DISCUSSION TITLE
Space for Website Statistics Requests

### DISCUSSION THREAD

**Discussion by Lachele** [GLYCAM-Web/website#233] [Ideas]
This is a space for making a wish list of places where we would like to keep website tracking data for reporting statistics and troubleshooting. 

We should make a format for this.  There might already be formats for this sort of thing.  

A request might look something like:

For each method of specifying a sequence (cb, lib, pb, txt, etc.), track:
- How many times the landing page (e.g., glycam.org/cb) is accessed
- How many times the visitor proceeds to the next page with bonus points for recording the pUUID
- How many times the visitor proceeds to the downloads page
- The number and type of files downloaded

**Comment by danwentworthart** [comment]
To assist with the last item on the list, these CB Download URLS each indicate some user clicked a button to get output. 

Download Minimized Structure only: `json/download/pdb/text/<str:gems_pUUID>/<str:conformer_ID>/`

Zip up just the default structure's pdb file: `json/download/pdb/zip/<str:pUUID>/`
Zip up project dir for one conformer: `json/download/conformer/<str:gems_pUUID>/<str:conformer_id>/`
Zip up a files for Unminimized Solvated Structure, specific water model: `json/download/conformer/solvated/<str:gems_pUUID>/<str:conformer_id>/<str:water_model>/`

Zip up all dirs (One level down, not recursive) requested in a typed project(would only be used by md or cb so far): `json/download/project/<str:app>/<str:gems_pUUID>/ 
Zip up unminimized-t3p.parm7 & 10.produ.nc only: `json/download-sim/<str:pUUID>/`

JSON_API-specific (no website) request for default for a given sequence: `download/sequence/<str:app>/<str:pUUID>/`
JSON_API-specific (no website) request for specific sequence/conformer: `download/sequence/<str:app>/<str:pUUID>/<str:conformerID>/`
